server/task_executor_pitr_restore.go

```diff
@@ -99,16 +100,18 @@ func (exec *PITRRestoreTaskExecutor) doPITRRestore(ctx context.Context, task *ap
 	log.Debug("Getting latest backup before or equal to targetTs...", zap.Time("targetTs", time.Unix(targetTs, 0)))
 	backup, err := mysqlRestore.GetLatestBackupBeforeOrEqualTs(ctx, backupList, targetTs)
 	if err != nil {
-		log.Error("Failed to get backup before or equal to targetTs",
+		dateTime := common.FormatDateTime(targetTs)
+		log.Error("Failed to get backup before or equal to time",
+			zap.String("dateTime", dateTime),
 			zap.Time("targetTs", time.Unix(targetTs, 0)),
 			zap.Error(err))
-		return fmt.Errorf("failed to get latest backup before or equal to targetTs[%d], error[%w]", targetTs, err)
+		return fmt.Errorf("failed to get latest backup before or equal to %s, error: %w", dateTime, err)
 	}
 	log.Debug("Got latest backup before or equal to targetTs", zap.String("backup", backup.Name))
 	backupFileName := getBackupAbsFilePath(dataDir, backup.DatabaseID, backup.Name)
 	backupFile, err := os.OpenFile(backupFileName, os.O_RDONLY, os.ModePerm)
 	if err != nil {
-		return fmt.Errorf("failed to open backup file[%s], error[%w]", backupFileName, err)
+		return fmt.Errorf("failed to open backup file: %s, error: %w", backupFileName, err)
```

Comment:
IMO colons in the middle of a sentence is not cool.

Reply:
@candy2255 's comment in the linear issue looks decent "Failed to get the latest backup no later than 2022-06-14 10:03:26. The target restore time 2022-06-14 10:03:26 is earlier than the oldest backup time 2022-06-14 10:04:00"

Reply:
@tianzhou IMHO it's a bit verbose, how about this:
"Failed to get the correct backup. The target restore time 2022-06-14 10:03:26 is earlier than the oldest backup time 2022-06-14 10:04:00"

We do not need to express our >= policy as is, because the user could tell from the last sentence that he/she is restoring to a time without a valid backup.

Reply:
`Correct seems confused`. Can you provide another word? How about valid?

Reply:
@d-bytebase More about the colons.
IMO, `"Reading %s: %v", filename, err` is cool, because the only colon is the last part of the sentence.
But `"Reading: %s, error: %v", filename, err` is not cool, because too many colons splits up the sentence into parts, and it's hard for eye balls to parse the range of each message for each colon.

Reply:
Rules of thumb:
1) follow what style says.
2) consistent with the rest of code.

Other discussions are subjective and don't bring much value.

Consider using errors.Wrapf() if that's better choice.